app/controllers/api/v1/search_results_controller.rb

```diff
@@ -19,8 +14,26 @@ def create
         end
       end
 
+      def index
+        # TODO: Add pundit to filter search results by user
+        search_result_list = SearchResult.where(user_id: current_user.id).order(:id)
+        pagy, paginated_results = paginated_resources_for(search_result_list)
+
+        search_result_serializer = SearchResultsSerializer.new(paginated_results, meta: meta_from_pagy(pagy))
```

Comment:
Do you it would be a good idea to refactor it with `render_success`? 💭

Reply:
@Tuubz Sounds good 👍 I will refactor `render_failed` to use `error_handler` instead